llvm/test/tools/sycl-post-link/spec-constants/spec_const_and_split.ll

```diff
@@ -2,27 +2,48 @@
 ; device code splitting and specialization constant processing are
 ; requested.
 ;
+; Additionally this test checks that sycl-post-link creates specialization
+; constant property correctly: i.e. it doesn't put all specialization constants
+; into properties of each device image
+
 ; RUN: sycl-post-link -split=kernel -spec-const=rt -S %s -o %t.files.table
-; RUN: FileCheck %s -input-file=%t.files_0.ll --check-prefixes CHECK0,CHECK
-; RUN: FileCheck %s -input-file=%t.files_1.ll --check-prefixes CHECK1,CHECK
+; RUN: FileCheck %s -input-file=%t.files_0.ll --check-prefixes CHECK-IR0,CHECK-IR
+; RUN: FileCheck %s -input-file=%t.files_1.ll --check-prefixes CHECK-IR1,CHECK-IR
+; RUN: FileCheck %s -input-file=%t.files_0.prop --check-prefixes CHECK-PROP0
+; RUN: FileCheck %s -input-file=%t.files_1.prop --check-prefixes CHECK-PROP1
 
 @SCSymID = private unnamed_addr constant [10 x i8] c"SpecConst\00", align 1
-; CHECK-NOT: @SCSymID
+@SCSymID2 = private unnamed_addr constant [11 x i8] c"SpecConst2\00", align 1
+; CHECK-IR-NOT: @SCSymID
+; CHECK-IR-NOT: @SCSymID
```

Comment:
Is this check duplicated intensionally?

Reply:
```suggestion
; CHECK-IR-NOT: @SCSymID2
```

No, it was supposed to check the second global variable for symbolic id

Reply:
Fixed in 3c7608bb910dcc923c2a1a6401404435a0976264